### ISSUE TITLE
View videos in gallery

### ISSUE THREAD

**Issue by mihaelmilea** [ubports/gallery-app#28]
Right now when you hit play for a video in gallery it opens the video in a video player that needs to be closed afterwards by pressing an "X". It would be great if the video can be played directly in the gallery-app and then you can swipe left or right to the previous or next photo or video without even pausing the played video. For example in the videos scope there is a choice to play the videos right there in the scope or to open them separately in the video player app.

**Comment by TronFourtyTwo** [comment]
Yes, the current one is an annoying behaviour. A built-in player would be better.
